application/libraries/services/NotificationService.php

```
@@ -0,0 +1,283 @@
+        $this->CI->load->model('user_m');
+    }
+
+    /**
+     * 댓글 알림 생성 (내 게시글에 댓글)
+     *
+     * @param int $articleId 게시글 ID
+     * @param int $commentId 댓글 ID
+     * @param int $actorId 댓글 작성자 ID
+     * @return void
+     */
+    public function notifyNewComment($articleId, $commentId, $actorId)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$commentId'. (undefined)

(UnusedFormalParameter)